src/main/java/com/mercadopago/client/payment/PaymentPayerRequest.java

```diff
@@ -4,10 +4,22 @@
 import lombok.Builder;
 import lombok.Getter;
 
+import java.util.Date;
+
 /** PaymentPayerRequest class. */
 @Getter
 @Builder
 public class PaymentPayerRequest {
+
+  /** Payer's authentication type. */
+  private final String authenticationType;
+
+  /** Payer's name. */
+  private final  String name;
```

Comment:
[req] Aqui sobrou um espaço desnecessário
```suggestion
  private final String name;
```